src/components/CommentList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import Comment from "./Comment";
+import { fetchArticleComments } from "./utilities/requests";
+import { indicateLoading } from "./utilities/utilityFunctions";
+
+export default function CommentList({ article_id }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchArticleComments(article_id).then((commentsObj) => {
+      setIsLoading(false);
+      setComments(commentsObj);
+    });
+  }, [article_id]);
+
+  if (isLoading) {
+    return indicateLoading();
+  }
+
+  return (
+    <section id="comments-section" className="text-center">
+      <h1>Comments</h1>
```

Comment:
```js
<h1>Comments</h1>
```
This probably isn't the main page title so shouldn't be a `h1`. The heading should descend in order so:
h1 - The single heading of the whole page
h2 - subheadings under that
h3 - subheadings under h2's
etc